packages/react-components/react-swatch-picker-preview/src/components/SwatchPicker/useSwatchPicker.ts

```diff
@@ -11,24 +14,48 @@ import type { SwatchPickerProps, SwatchPickerState } from './SwatchPicker.types'
  * @param props - props from this instance of SwatchPicker
  * @param ref - reference to root HTMLElement of SwatchPicker
  */
-export const useSwatchPicker_unstable = (
+export const useSwatchPicker_unstable = <T>(
```

Comment:
```suggestion
export const useSwatchPicker_unstable = (
```

I don't see `T` being used